packages/ripple-nuxt-tide/lib/core/tide.js

```diff
@@ -98,7 +98,8 @@ export const tide = (axios, site, config) => ({
   getMenuFields: function () {
     return {
       menuMain: 'field_site_main_menu',
-      menuFooter: 'field_site_footer_menu'
+      menuFooter: 'field_site_footer_menu',
+      menuMainFooter: 'field_site_footer_main_menu'
```

Comment:
Hi @awset
Can we put in a condition here?
If field_site_footer_main_menu is not available, we use field_site_footer_menu as default?

Reply:
no we can't, the condition is available here, https://github.com/dpc-sdp/ripple/pull/1086/files#diff-ee87bd1376d57e8223e58511cb4dfa714906292d41a23013d1e714334c6e36cfR71

Reply:
Ah - thank you!